bigbluebutton-html5/imports/ui/components/presentation/presentation-toolbar/component.jsx

```diff
@@ -165,13 +165,15 @@ class PresentationToolbar extends PureComponent {
     } = this.props;
 
     if (event) event.currentTarget.blur();
+    this.props.endCurrentPoll();
```

Comment:
please  add `endCurrentPoll` to the list of items we get from `this.props` (2 lines above)